client/components/suggestion/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import omit from 'lodash/omit';
+import FormLabel from 'components/forms/form-label';
+import FormRadio from 'components/forms/form-radio';
+import Button from 'components/button';
+import Summary from 'components/summary';
+import { translate as __ } from 'lib/mixins/i18n';
+
+const RadioButton = ( props ) => {
+	return (
+		<FormLabel>
+			<FormRadio { ...omit( props, 'children' ) } />
+			{ props.children }
+		</FormLabel>
+	);
+};
+
+const Suggestion = ( { acceptSuggestion, formValues, formActions, layout, suggestions } ) => {
+	return (
+		<div>
+			<RadioButton
+				checked={ ! acceptSuggestion }
+				onChange={ () => formActions.setFormProperty( 'acceptSuggestion', false ) } >
+				{ layout.suggestion_original_title }
+				<Summary
+					formValues={ formValues }
+					layoutItems={ layout.items }
+					summaryTemplate={ layout.template } />
+				<Button onClick={ formActions.backFromSuggestion } >
+					{ __( 'Edit' ) }
+				</Button>
+			</RadioButton>
+			<RadioButton
+				checked={ acceptSuggestion }
+				onChange={ () => formActions.setFormProperty( 'acceptSuggestion', true ) } >
+				{ layout.suggestion_corrected_title }
+				<Summary
+					formValues={ formValues }
+					layoutItems={ layout.items }
+					summaryTemplate={ layout.template }
+					overrideFields={ suggestions } />
+			</RadioButton>
+		</div>
+	);
+};
+
+Suggestion.propTypes = {
+	layout: PropTypes.shape( {
+		template: PropTypes.string,
+		items: PropTypes.array,
+		suggestion_original_title: PropTypes.string,
+		suggestion_corrected_title: PropTypes.string,
```

Comment:
Instead of having these two props, what if we had an array of suggestions+titles for our address picker. At some point in MVP we will probably need to support choosing from among > 2 addresses (think about amazon's checkout where it shows multiple addresses and asks you to pick one.)